tutorials/s4sdk-secure-cloudfoundry/s4sdk-secure-cloudfoundry.md

```diff
@@ -90,168 +69,150 @@ This alternative approach requires that you have npm installed on your machine.
     }
     ```
 
-3. Install AppRouter dependencies using the following commands
-
+3. Within `<destLocation>/approuter` create a new file called `xs-app.json` with the following content:
+    ```json
+    {
+      "welcomeFile": "index.html",
+      "routes": [{
+        "source": "/",
+        "target": "/",
+        "destination": "app-destination"
+        }]
+    }
     ```
-    npm config set @sap:registry https://npm.sap.com
-    npm install
+
+4. Last but not least create a new `manifest.yml` file within `<destLocation>` for the app router microservice with the following content:
+
+    ```yaml
+
+    ---
+    applications:
+    - name: approuter
+      routes:
+        - route: approuter-<subdomain>.cfapps.<region_id>.hana.ondemand.com
+      path: approuter
+      memory: 128M
+      buildpacks:
+        - nodejs_buildpack
+      env:
+        TENANT_HOST_PATTERN: 'approuter-(.*).cfapps.<region_id>.hana.ondemand.com'
+        destinations: '[{"name":"app-destination", "url" :<APPLICATION_URL>, "forwardAuthToken": true}]'
+      services:
+        - my-xsuaa
     ```
 
-[DONE]
-[ACCORDION-END]
+    Adapt the file as follows:
 
-[ACCORDION-BEGIN [Step 4: ](Proceed with the App Router Setup independent of your download preferences)]
-1. Within **<destLocation>/approuter** create a new file called **`xs-app`.json** with the following content:
-```
-{
-  "welcomeFile": "index.html",
-  "routes": [{
-    "source": "/",
-    "target": "/",
-    "destination": "app-destination"
-  }]
-}
-```
+      1. Replace `<subdomain>` with your subdomain. You will find your subdomain in the CF cockpit by heading to the overview page of your sub-account:
 
-2. Within <destLocation> create a new **`manifest.yml`** file for the AppRouter microservice with the following content.
+          ![Subomain and Tenant ID in the CF Cockpit](Figure-3-1.png)
```

Comment:
This image needs to show the Subdomain, but it appears to be showing part of a tenant ID.

Reply:
The subdomain and most parts of the tenant ID are blurred out in order to not give away account details.